helm/credentiald-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: credentiald-network-policy
+  namespace: giantswarm
+spec:
+  podSelector:
+    matchLabels:
+      app: credentiald
+  policyTypes:
+  - Ingress
+  - Egress
+  # Allow access only from api server.
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          app: api
+    ports:
+    - protocol: TCP
+      port: 8000
+  # Allow all egress traffic.
```

Comment:
Does `credentiald` need all egress? I feel like it just needs to talk to api

Reply:
there is no elegant way to allow only kubernetes API. because kubernetes API is not a pod (it's servive that points to real vm IP) and we don't have static IP address for it (IP ranges will change).

So i updated and now i allow all DNS and HTTPS traffic. Please take a look.